source/simulator/assemblers/stokes.cc

```diff
@@ -447,7 +447,15 @@ namespace aspect
       internal::Assembly::CopyData::StokesSystem<dim> &data = dynamic_cast<internal::Assembly::CopyData::StokesSystem<dim>& > (data_base);
 
       // assemble RHS of:
-      //  - div u = 1/rho * drho/dp rho * g * u
+      //  - div \mathbf{u} = \frac{1}{\rho} \frac{\partial rho}{\partial p} \rho \mathbf{g} \cdot \mathbf{u}
+      // to the manual, this term seems to have the wrong sign, but this
+      // is because we negate the entire equation to make sure we get
+      // -div(u) as the adjoint operator of grad(p)
+
+      // Compared to the manual, this term seems to have the wrong sign, but
+      // this is because we negate the entire equation to make sure we get
+      // -div(u) as the adjoint operator of grad(p)
```

Comment:
Please remove the duplicate documentation (copy and paste problem). Note that the first version is missing the word "compared".